app/models/user.rb

```diff
@@ -0,0 +1,4 @@
+class User < ApplicationRecord
+    has_secure_password
+    has_many :followers, class_name: 'User'
```

Comment:
@idprietop why a **class_name** is needed here?

Reply:
I also forgot to remove this, this line is not part of this PR.

Reply:
ok!